README.md

```diff
@@ -4,10 +4,20 @@ A gRPC interceptor to mask errors.
 
 This piece of middleware intends to stop sensitive information from being returned to calling services. Pass in a list of codes you wish to mask, and they shall all be returned as internal server errors, with no further information.
 
+## Go get
+
+```sh
+go get github.com/tumelohq/grpc-middleware/...
+```
+
 ## Usage 
 
 Have a look at the test, we spin up an entire client and server, and plug the middleware in.
```

Comment:
It might be worth just pointing to the docs, possibly with a nice cheeky badge. https://github.com/badges/shields/issues/81